lib/pat.c

```diff
@@ -5876,25 +5885,26 @@ search_push(grn_ctx *ctx,
       }
     }
   } else {
-    pat_node *pn;
+    pat_node_common *pn;
```

Comment:
Should we use the full name here for readability?
```suggestion
    pat_node_common *pat_node;
```

Reply:
I think `pat_` prefix is redundant.
Is it ok if I will write `node` in this variable name instead of `pat_node`?

Reply:
Hmm...
Is pn really abbreviation of `pat_node`?

Reply:
I think it is. Could you share why you think of it with me?

Reply:
`parent node`...?